code/datums/diseases/advance/symptoms/lesserzombie.dm

```diff
@@ -0,0 +1,54 @@
+/datum/disease/lesserzombie
+	name = "Andromeda blobitis"
+	max_stages = 15
+	spread_text = "Consumption of or contact with invasive organic material"
+	cure_text = "Phenol and Mannitol, or Atropine"
+	cures = list("phenol", "mannitol")
+	cures = list("atropine")

```

Comment:
If you set a field to a variable twice, which was inherited, the last value that BYOND reads will be set to that. In this case, BYOND will read line 6, set the value, and then read line 7 and replace what was set in line 6.